notebooks/official/tabular_workflows/tabnet_on_vertex_pipelines.ipynb

```diff
@@ -0,0 +1,1287 @@
+{
```

Comment:
<p>Is there no prediction step in this notebook?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/726/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='49'/>

Reply:
No, that's out of scope for this notebook.

Reply:
<ul><li>I think you should state this explicitly in the introduction. </li><li>What's the next step for people who create one of these pipelines and want to make a prediction? </li></ul>

Reply:
<p>Sure, added it to the Objective. This code cell returns a link to the model UI. The customer can click on it and from there, deploy the model, do batch prediction etc. I updated the cell above to mention this.</p>